testsuite/integration/clustering/pom.xml

```diff
@@ -113,26 +114,20 @@
         <dependency>
             <groupId>org.infinispan</groupId>
             <artifactId>infinispan-server-testdriver-junit4</artifactId>
-            <version>${version.org.infinispan}</version>
+            <version>${version.org.infinispan.server.driver}</version>
             <exclusions>
+                <!-- Avoids issue on deploy - "Caused by: org.jboss.as.cli.CommandRegistry$RegisterHandlerException: The following command names could not be registered since they conflict with the already registered ones: cd" -->
                 <exclusion>
                     <groupId>org.infinispan</groupId>
-                    <artifactId>infinispan-server-runtime</artifactId>
+                    <artifactId>infinispan-cli-client</artifactId>
                 </exclusion>
             </exclusions>
         </dependency>
+        <!-- Avoids REST call timeout on cluster shutdown -->
         <dependency>
-            <groupId>org.infinispan</groupId>
-            <artifactId>infinispan-client-rest</artifactId>
-            <version>${version.org.infinispan}</version>
-        </dependency>
-        <dependency>
-            <groupId>org.infinispan</groupId>
-            <artifactId>infinispan-commons</artifactId>
-        </dependency>
-        <dependency>
-            <groupId>org.infinispan</groupId>
-            <artifactId>infinispan-client-hotrod</artifactId>
+            <groupId>com.squareup.okhttp3</groupId>
+            <artifactId>okhttp</artifactId>
+            <version>3.14.8</version>
```

Comment:
Create a property for this version.

Reply:
Do we need such a property? There is already a property for com.squareup.okhttp3 however its using an outdated version at the moment, hopefully this will eventually go away when jaegertracing uses a later version.

Reply:
This was the properties (and their intent) are more visible.  This pom can override the property with no problem, no?

Reply:
I see what you mean now, removed the dependency and just override the property and it works. Originally I didn't realize we already manage this dependency in the main pom.xml; fixed now.